src/Sylius/Component/Core/OrderProcessing/StateResolver.php

```diff
@@ -37,12 +37,17 @@ public function resolvePaymentState(OrderInterface $order)
             foreach ($payments as $payment) {
                 if (PaymentInterface::STATE_COMPLETED === $payment->getState()) {
                     $completedPaymentTotal += $payment->getAmount();
+                } elseif (PaymentInterface::STATE_REFUNDED == $payment->getState()) {
+                    $completedPaymentTotal -= $payment->getAmount();
```

Comment:
Actually, I think we do not have to subtract it, cause it counts only completed payments, hm? If payment state changes to refunded, then it is not completed. Unless you want to create a new payment for refund? I was thinking about create a separate model for that.